test/object-store/CMakeLists.txt

```diff
@@ -126,36 +126,23 @@ target_include_directories(ObjectStoreTests PRIVATE
 # on Apple platforms we use the built-in CFRunLoop
 # everywhere else it's libuv, except UWP where it doesn't build
 if(NOT APPLE AND NOT WINDOWS_STORE)
-    option(REALM_FETCH_MISSING_DEPENDENCIES "Download missing dependencies with CMake's FetchContent where possible" ON)
-    if(REALM_FETCH_MISSING_DEPENDENCIES)
```

Comment:
What is the reason behind removing this option?

Reply:
We change the CMake file, dependencies are always fetched if needed. Check the rest of the cmake file.